cnxpublishing/views/__init__.py

```diff
@@ -1 +1,70 @@
 # -*- coding: utf-8 -*-
+from pyramid.httpexceptions import default_exceptionresponse_view
+
+
+def declare_api_routes(config):
+    """Declaration of routing"""
+    add_route = config.add_route
+    add_route('get-content', '/contents/{ident_hash}')
+    add_route('get-resource', '/resources/{hash}')
+
+    # User actions API
+    add_route('license-request', '/contents/{uuid}/licensors')
+    add_route('roles-request', '/contents/{uuid}/roles')
+    add_route('acl-request', '/contents/{uuid}/permissions')
+
+    # Publishing API
+    add_route('publications', '/publications')
+    add_route('get-publication', '/publications/{id}')
+    add_route('publication-license-acceptance',
+              '/publications/{id}/license-acceptances/{uid}')
+    add_route('publication-role-acceptance',
+              '/publications/{id}/role-acceptances/{uid}')
+    # TODO (8-May-12017) Remove because the term collate is being phased out.
+    add_route('collate-content', '/contents/{ident_hash}/collate-content')
+    add_route('bake-content', '/contents/{ident_hash}/baked')
```

Comment:
`baked` is not a verb. Does this route do the baking, or return the baked version?